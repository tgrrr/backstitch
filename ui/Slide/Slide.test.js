import React from 'react'
import renderer from 'react-test-renderer'
import { Slide } from '.'

it('Slide: default', () => {
    const component = renderer.create(
        <Slide>
            <>
                Slide
            </>
        </Slide>
    )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
