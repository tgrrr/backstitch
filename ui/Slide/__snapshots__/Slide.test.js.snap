// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Slide: default 1`] = `
<div
  className="sc-AxjAm StDqN"
>
  <div
    className="Slide "
    data-testid="Slide"
  >
    Slide
  </div>
</div>
`;
